server/index.js

```diff
@@ -15,7 +15,7 @@ app.use(bodyParser.json());
 
 app.use((req, res, next) => {
   const timestamp = new Date();
-  console.log(`serving a ${req.method} request to url ${req.url} at ${timestamp.getHours()}:${timestamp.getMinutes()}:${timestamp.getSeconds}. Body: `, req.body);
+  console.log(`serving a ${req.method} request to url ${req.url} at ${timestamp.getHours()}:${timestamp.getMinutes()}:${timestamp.getSeconds()}. Body: `, req.body);
```

Comment:
This is awesome! Not sure if it was your doing but it's a great idea.

Reply:
Thanks! Robin suggested this in an office hour one time and I've been writing it into all my servers ever since. Super super helpful